src/main/java/bio/terra/service/snapshot/SnapshotService.java

```diff
@@ -282,6 +283,11 @@ void validateForByRequestIdMode(SnapshotRequestContentsModel snapshotRequestCont
       throw new ValidationException(
           "The createdBy email supplied on the access request is not valid.");
     }
+    if (snapshotRequest.getDataAccessControlGroups() != null
+        && !snapshotRequest.getDataAccessControlGroups().isEmpty()) {
+      throw new ValidationException(
+          "Data access control groups are not allowed for snapshots created byRequestId.");
```

Comment:
This is because we only allow one data access control group, right?

Reply:
https://github.com/DataBiosphere/jade-data-repo/pull/1742#discussion_r1683155396